frontend/javascripts/oxalis/view/action-bar/toolbar_view.tsx

```diff
@@ -443,16 +452,17 @@ function AdditionalSkeletonModesButtons() {
           alt="Merger Mode"
         />
       </ButtonComponent>
-      {features().jobsEnabled && isMergerModeEnabled && (
-        <ButtonComponent
-          style={NARROW_BUTTON_STYLE}
-          onClick={() => setShowMaterializeVolumeAnnotationModal(true)}
-          title="Materialize this merger mode annotation into a new dataset."
-        >
-          <ExportOutlined />
-        </ButtonComponent>
-      )}
-      {features().jobsEnabled && showMaterializeVolumeAnnotationModal && (
+      {isMergerModeEnabled &&
+        isMaterializeVolumeAnnotationEnabled(
```

Comment:
this is called as a function but it's a boolean?

Reply:
Good catch. This is wrong